catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/security/Constants.java

```diff
@@ -27,7 +27,7 @@
           WorkspaceAttributes.WORKSPACE_TAG,
           QueryTemplateType.QUERY_TEMPLATE_TAG);
 
-  public static final String IS_SHAREABLE = "security.tag.is-shareable";
+  public static final String SYSTEM_TEMPLATE = "System Template";
```

Comment:
❓  Wasn't a large part of the previous PR to generalize metacard sharing? I feel like `security.tag.is-shareable` is much more broadly applicable than a `System Template` String.

Maybe we could have both?

Reply:
@mackncheesiest IS_SHAREABLE actually will not exist anymore based on the sharing changes since the purpose of that tag previously is now inferred via SECURITY_ATTRIBUTES in Core. 

System Template is more or less only there to be assigned to metacards without an owner during initialization.